server/models/Profile.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+
```

Comment:
#40 @abhkoirala has almost completed Profile model. So, I think we don't need Profile model here.